pom.xml

```diff
@@ -15,8 +15,28 @@
 	<url>http://github.com/qbicsoftware/statistics-portlet</url>
 	<description>Portlet to visualize data on landing page</description>
 	<packaging>war</packaging>
+	<properties>
+		<log4j.version>2.15.0</log4j.version>
+	</properties>
 	<!-- we only need to tell maven where to find our parent pom and other QBiC dependencies -->
 	<repositories>
+		<repository>
+			<id>vaadin-addons</id>
+			<url>https://maven.vaadin.com/vaadin-addons</url>
+		</repository>
+		<repository>
+			<releases>
+				<enabled>true</enabled>
+				<updatePolicy>always</updatePolicy>
+				<checksumPolicy>fail</checksumPolicy>
+			</releases>
+			<snapshots>
+				<enabled>false</enabled>
+			</snapshots>
+			<id>maven-central</id>
+			<name>Maven central</name>
+			<url>https://repo.maven.apache.org/maven2</url>
+		</repository>
```

Comment:
I think you should add them at the end of the list so our nexus repo is searched first